assets/src/blocks/Articles/ArticlesEditor.js

```diff
@@ -153,7 +153,7 @@ const renderView = ({ attributes, postType, posts, totalPosts }, toAttribute) =>
 export const ArticlesEditor = (props) => {
   const { isSelected, attributes, setAttributes } = props;
 
-  const { postType, postId } = useSelect((select) => ({
+  const { postType, postId } = ((select) => ({
```

Comment:
Heads up! this looks like an accident ☝️

Reply:
Oops indeed! 😱